buildSrc/src/main/kotlin/BuildExtensions.kt

```diff
@@ -197,28 +199,35 @@ fun Project.configureTaskDependencies() {
     }
 
     afterEvaluate {
+        val launchProtoData = "launchProtoData"
```

Comment:
Speaking of Dokka. I think when its time comes, this is going to be the first reason why our Dokka-based stuff is most likely broken right now.

Just noticed.